src/NuGet.Core/NuGet.Packaging/Signing/Verification/SignatureTrustAndValidityVerificationProvider.cs

```diff
@@ -65,12 +64,13 @@ public SignatureTrustAndValidityVerificationProvider(IEnumerable<KeyValuePair<st
             var status = SignatureVerificationStatus.Unknown;
             var issues = Enumerable.Empty<SignatureLog>();
             var isUntrustedRootAllowed = IsUntrustedRootAllowed(signature);
+            var isUnknownRevocationAllowed = RuntimeEnvironmentHelper.IsLinux && isUntrustedRootAllowed;
```

Comment:
This is to account that on Linux UntrustedRoot will always also return unknown revocation. If this is incorrect we should revert this change.